pkg/apiserver/handlers.go

```diff
@@ -188,11 +190,11 @@ func (t *timeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	done := make(chan struct{}, 1)
+	done := make(chan struct{})
 	tw := newTimeoutWriter(w)
 	go func() {
 		t.handler.ServeHTTP(tw, r)
```

Comment:
No-- we now deliberately bring down the entire process on a panic, this is something that changed a few weeks ago. I mean, you could `defer kruntime.HandlePanics()` or whatever but it now re-panics after catching panics.